IPython/core/formatters.py

```diff
@@ -139,6 +139,9 @@ def format(self, obj, include=None, exclude=None):
             precedence and the corresponding `_repr_*_` for this mimetype will
             not be called.
 
+            TODO: Figure out and document whether user registered formatters do
```

Comment:
Yes, in theory it seem to not be the case with `_repr_mimebundle_`, I should have been clearer in the comment.